startscripts/hello_world

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
```

Comment:
I think this file should stay for documentation and quick testeing this functioanlity (includes autocomplete)